### ISSUE TITLE
Naming/AccessorMethodName not flagging all functions in some ruby files, even when explicitly included

### ISSUE THREAD

**Issue by rnickmccandless** [rubocop-hq/rubocop#7801]
The cop "Naming/AccessorMethodName" is not flagging all functions that start with `get_` and `set_` in some ruby files, even when explicitly including the files in the rubocop config.

We want to discourage our coders from using any function starting with `get_` and `set_` by including all files within the ruby app.

```yaml
Naming/AccessorMethodName:
  Include:
    - 'app/**/*.rb'
```

--------

## Expected behavior

Rubocop flagging all functions that start with starting with `get_` and `set_` in all ruby files that are added to the `Include` in the "Naming/AccessorMethodName" rubocop config.

## Actual behavior

Rubocop flags functions that start with `get_` and `set_` _only_ in certain ruby files such as controllers and modes while ignoring files added to the `Include`. 

## Steps to reproduce the problem

Add the following rule to the `.rubocop.yml`

```yaml
Naming/AccessorMethodName:
  Include:
    - 'app/**/*.rb'
```

Create a function starting with `get_something_from(id)` in `app/services/external_source.rb`

Run rubocop 

```bash
bundle exec rubocop
```

## RuboCop version

```
$ [bundle exec] rubocop -V
0.80.1 (using Parser 2.7.0.4, running on ruby 2.6.2 x86_64-darwin19)
```

**Comment by Kache** [comment]
I ran into something like this, I believe the cop does not treat singleton accessor methods (module and class methods) as offenses, since singleton methods in principle should be free functions and not bound to an object instance.

For example:

```ruby
module Foo
  def get_this_gets_caught
    'foo'
  end

  def self.get_this_does_not
    'foo'
  end
end
```

**Comment by dvandersluis** [comment]
This cop is working as expected. Both instance and class methods are considered, and it is not limited to certain paths.

```
test.rb:2:7: C: Naming/AccessorMethodName: Do not prefix reader method names with get_.
  def get_this_gets_caught
      ^^^^^^^^^^^^^^^^^^^^
test.rb:6:12: C: Naming/AccessorMethodName: Do not prefix reader method names with get_.
  def self.get_this_does_not
           ^^^^^^^^^^^^^^^^^
```

However, the cop only checks for methods with the correct arity: getters must have 0 arguments and setters must have 1. I am opening a PR to update the documentation in that regard.
